libraries/libstratosphere/include/stratosphere/lm/detail/lm_detail_log_types.hpp

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2018-2020 Atmosphère-NX
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms and conditions of the GNU General Public License,
+ * version 2, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include <stratosphere/diag/diag_log_types.hpp>
+
+namespace ams::lm::detail {
+
+    enum LogPacketFlags {
+        LogPacketFlags_Head         = (1 << 0),
+        LogPacketFlags_Tail         = (1 << 1),
+        LogPacketFlags_LittleEndian = (1 << 2)
+    };
+
+    struct BinaryLogHeader {
+        static constexpr u32 Magic = util::FourCC<'h','p','h','p'>::Code;
```

Comment:
Is this the correct endianness?

Reply:
Given the code is conceptually "phph" (packet header, packet header), may well do to use reverse code with that encoding.